### DISCUSSION TITLE
Empty json from navigation endpoint

### DISCUSSION THREAD

**Discussion by SuperFlySly** [sulu/SuluHeadlessBundle#142] [Q&A]
Hi guys, 

when I try to fetch the Navigation, I always get this result:

```
{
	"_embedded": {
		"items": []
	}
}
```

any idea?

**Comment by alexander-schranz** [comment]
Do you only get empty items over the headless API? Does the Sulu Twig extensions still give you your configured pages? I think you need provide here more information. Which navigation context you created in your webspace, which pages and where in your tree structure you did add them to that navigation context and what the Full URL to fetch the navigation API is.

**Comment by Ph0xEn** [comment]
I had the same issue. Try to use the depth=2 parameter in you url.
